### PR TITLE
Feat: add blocks for reading from serial, add writing to serial monitor

### PR DESCRIPTION
**Pull request by sverben**
This PR relies on https://github.com/leaphy-robotics/leaphy-blocks/pull/41

### CODE REVIEW

File: src/app/services/arduino-uploader/ArduinoUploader.ts [hunk 1]
```diff
@@ -391,21 +391,22 @@ class Arduino {
...
      */
-    async serialMonitor(writableStream: WritableStream) {
+    async serialMonitor(outputStream: WritableStream) {
         if (this.robotWiredState.getSerialPort() == null) {
             return;
         }
         this.port = this.robotWiredState.getSerialPort();
 
         try {
+            console.log(this.port)
```
Comment (rmoesbergen) [inline src/app/services/arduino-uploader/ArduinoUploader.ts:403]:
```suggestion
```

**Comment by koen1711** [comment]
waiting for review until other PR is approved and merged

**Comment by koen1711** [comment]
bump @leaphy-robotics/leaphy-blocks version please

**Review by koen1711 — APPROVED** [review]
also bump package.json version please

**Pull request merged**